[DOCS] update release links in Libraries (#2236)
open-edge-platform/edge-ai-libraries: microservices/time-series-analytics/README-dockerhub.md

It is a powerful, flexible solution for real-time analysis of time series data. Built on top of Kapacitor, it enables both streaming and batch processing, seamlessly integrating with InfluxDB for efficient data storage and retrieval.

> **Note**:
Please note this prebuilt docker image is not intended for production. For production use, build the docker image yourself using the [Dockerfile](https://github.com/open-edge-platform/edge-ai-libraries/blob/release-2026.1.0/microservices/time-series-analytics/Dockerfile)
by following these [build instructions](https://docs.openedgeplatform.intel.com/2026.1/edge-ai-libraries/time-series-analytics/get-started.html#build-docker-image).

## Supported versions
---
> **Note**: The tags suffixed with `-weekly` and `-rcX` are developmental builds that may not be stable.

> ### [1.1.0](https://docs.openedgeplatform.intel.com/dev/edge-ai-libraries/time-series-analytics/release-notes/release-notes-2025.html)

Reviewer: ```suggestion
### [1.1.0](https://docs.openedgeplatform.intel.com/2026.1/edge-ai-libraries/time-series-analytics/release-notes/release-notes-2025.html)
```

#### Deploy using Docker Compose
---

> For more details on deployment, refer to the [documentation](https://docs.openedgeplatform.intel.com/2026.1/edge-ai-libraries/time-series-analytics/get-started.html).

Reviewer: ```suggestion
For more details on deployment, refer to the [documentation](https://docs.openedgeplatform.intel.com/2025.2/edge-ai-libraries/time-series-analytics/get-started.html).
```

This one remains unchanged, since H3 in L13 above is referring to version 1.1.0 (i.e. the 2025.2 release).

#### Deploy on Kubernetes cluster using Helm Charts
---

> For more details on deployment, refer to the [documentation](https://docs.openedgeplatform.intel.com/2026.1/edge-ai-libraries/time-series-analytics/get-started/deploy-with-helm.html).

Reviewer: ```suggestion
For more details on deployment, refer to the [documentation](https://docs.openedgeplatform.intel.com/2025.2/edge-ai-libraries/time-series-analytics/get-started/deploy-with-helm.html).
```

As above, this is still in reference to a previous version (v 1.1.0/release 2025.2).

### [1.0.0](https://docs.openedgeplatform.intel.com/2025.1/edge-ai-libraries/time-series-analytics/release-notes/aug-2025.html#v1-0-0)

#### Deploy using Docker Compose
---

> For more details on deployment, refer to the [documentation](https://docs.openedgeplatform.intel.com/2026.1/edge-ai-libraries/time-series-analytics/get-started.html).

Reviewer: ```suggestion
For more details on deployment, refer to the [documentation](https://docs.openedgeplatform.intel.com/2025.1/edge-ai-libraries/time-series-analytics/get-started.html).
```

As above, but this time it's for the 2025.1 release.

#### Deploy on Kubernetes cluster using Helm Charts
---

> For more details on deployment, refer to the [documentation](https://docs.openedgeplatform.intel.com/2026.1/edge-ai-libraries/time-series-analytics/get-started/deploy-with-helm.html).

Reviewer: ```suggestion
For more details on deployment, refer to the [documentation](https://docs.openedgeplatform.intel.com/2025.1/edge-ai-libraries/time-series-analytics/get-started/deploy-with-helm.html).
```

As above.

---
> **Note**: The tags suffixed with `-weekly` and `-rcX` are developmental builds that may not be stable.

> ### [1.1.0](https://docs.openedgeplatform.intel.com/2026.1/edge-ai-libraries/time-series-analytics/release-notes/release-notes-2025.html)

Reviewer 2: ```suggestion
### [1.1.0](https://docs.openedgeplatform.intel.com/2025.2/edge-ai-libraries/time-series-analytics/release_notes/dec-2025.html)
```
